### ISSUE TITLE
Language not properly set in machine-generated repository created from O/MSS

### ISSUE THREAD

**Issue by davidzbiral** [DISSINET/InkVisitor#1538]
The value in repository_label_language should be the label language. Not properly set by the parsing. 
Observed in Import batch [development] 2023-01-13 19:35:45.939035.

Source data:
![obrazek](https://user-images.githubusercontent.com/55400794/213872580-651dbbf2-c48e-41ee-9755-920914a6d539.png)

Result: Latin.
![obrazek](https://user-images.githubusercontent.com/55400794/213872640-db288489-8b74-452e-8ae9-85e24c821bad.png)

**Comment by tomaham** [comment]
Yes, I know why this is, will be good after reimport (would be right even without reporting).
